### DISCUSSION TITLE
To test and use more things

### DISCUSSION THREAD

**Discussion by Simleybob** [magknight/787aviatorsTracker#561] [Ideas]
There are still some crucial things, which you can't test are working in pre-flight. An example of this is the 'Fire OVHT Test', the cargo fire arm also is not functional. The emergency lights would be a nice feature to able to lift the guard, turn off and on and arm them etc, and same with the ELT. Also the 'AIR COND RESET button is functional, but not animated. Furthermore the washers both left and right are not functional. Also the Stab norm guards, and you can't operate the trim with the mouse - this would be nice (like on the yoke and next to the parking brake), especially for those operating with a mouse or don't have it assigned on their joystick. Also you can't lock override currently or use the ATLN gear guard, which would be useful if one would need for failures. Sorry to be annoying lol.

**Comment by aeroniemi** [comment]
This is primarily dependent on significant art changes, which are planned for after the completion of the wings - likely will only support "normal" ops, not anything that is only used in non-normal operations

**Reply by Simleybob** [reply]
Okay thanks
